SunEngine.Commons/Cache/Services/ContentCache.cs

```diff
@@ -85,7 +77,13 @@ public void InvalidateCache(string categoryName)
             if (categoryId == null)
                 return;
 
-            InvalidateCache((int)categoryId);
+            InvalidateCache((int) categoryId);
```

Comment:
Could you clarify this action? We cast the string to int, is it okay to do so?

Reply:
It`s not string, It`s Nullable<int>